vendor/adminer/index.php

```diff
@@ -0,0 +1,179 @@
+<?php
+/**
+ * Adminer configuration for WordPress Studio.
+ */
+
+require_once( dirname( __FILE__ ) . '/config.php' );
+
+// Utils
+function wp_studio_escape_html( $string ) {
+	return htmlspecialchars( $string, ENT_QUOTES, 'UTF-8' );
+}
+
+/**
+ * Creates and returns the Adminer object with custom configuration.
+ *
+ * @return AdminerEditorCustomizations
+ */
+function adminer_object() {
+  // @see https://github.com/vrana/adminer/blob/master/editor/include/adminer.inc.php for overrideable methods.
+	class AdminerEditorCustomizations extends Adminer\Adminer {
+		/**
+		 * Modifies the login form field to default to SQLite.
+		 *
+		 * @param string $name    Field name.
+		 * @param string $heading Field heading.
+		 * @param string $value   Field value.
+		 * @return string
+		 */
+		public function loginFormField( $name, $heading, $value ) {
+			return parent::loginFormField( $name, $heading, str_replace( 'value="server"', 'value="sqlite"', $value ) );
+		}
+
+		/**
+		 * Returns the path to the SQLite database.
+		 *
+		 * @return string
+		 */
+		public function database() {
+			return ADMINER_SQLITE_DATABASE_PATH;
+		}
+
+		/**
+		 * Returns the custom name for the admin interface.
+		 *
+		 * @return string
+		 */
+		public function name() {
+			return 'WordPress Studio';
+		}
+
+		/**
+		 * Returns the server name display.
+         * Get server name displayed in breadcrumbs.
```

Comment:
Tiny formatting nit (spaces vs tabs). I'm not 100% sure the code suggestion here is correct but I copy/pasted from the previous line:

```suggestion
		 * Get server name displayed in breadcrumbs.
```

Reply:
Let's YOLO it 😄 Thanks!